control-plane/agents/src/bin/core/volume/operations.rs

```diff
@@ -563,22 +564,29 @@ impl ResourceShutdownOperations for OperationGuardArc<VolumeSpec> {
     ) -> Result<(), SvcError> {
         let shutdown_nexuses = registry
             .specs()
-            .get_volume_shutdown_nexuses(&request.uuid)
+            .get_volume_shutdown_nexuses(&request.uuid())
```

Comment:
How has this been working till now?

Reply:
uuid field in the request used to be public field. Now i have added a getter fn uuid(). Made the field private.